### PR TITLE
Command cleanup

### PR DESCRIPTION
**Pull request by funkychicken493**
A simple pull request consisting of formatting changes/fixes for the /denizen and /npc commands.

### CODE REVIEW

File: plugin/src/main/java/com/denizenscript/denizen/utilities/command/NPCCommandHandler.java [hunk 1]
```diff
@@ -367,7 +366,7 @@ else if (args.hasValueFlag("anchor")) {
                     return;
                 }
             }
-            Messaging.sendError(sender, "The NPC does not have the specified anchor!");
+            Messaging.sendError(sender, "NPC " + npc.getName() + " does not have the anchor '" + args.getFlag("anchor") + "'!");
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/utilities/command/NPCCommandHandler.java]: Test this with NPCs that have colored names and make sure it compensates appropriate (same for other lines you added name to)

File: plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java [hunk 2]
```diff
@@ -329,15 +333,15 @@ else if (args.getString(1).equalsIgnoreCase("scripts")) {
                 if (ScriptHelper.hadError()) {
                     Messaging.sendError(sender, "There was an error loading your scripts, check the console for details!");
                 }
-                Messaging.sendError(sender, "'/denizen reload scripts' is the old way of doing things ... use '/ex reload' instead!");
+                Messaging.sendError(sender, "'/denizen reload scripts' is deprecated, use '/ex reload' instead.");
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java]: Incorrect change

File: plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java [hunk 3]
```diff
@@ -279,7 +283,7 @@ public void version(CommandContext args, CommandSender sender) throws CommandExc
      */
     @Command(
             aliases = {"denizen"}, usage = "save",
-            desc = "Saves the current Denizen save data to file as needed.", modifiers = {"save"},
+            desc = "Saves the current Denizen save data to disk as needed.", modifiers = {"save"},
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java]: Incorrect change

File: plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java [hunk 4]
```diff
@@ -23,6 +23,10 @@
 
 public class DenizenCommandHandler {
 
+    public static final String DISCORD_LINK = "https://discord.gg/Q6pZGSR";
+    public static final String WEBSITE_LINK = "https://denizenscript.com";
+    public static final String GUIDE_LINK = "https://guide.denizenscript.com/";
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java]: This shouldn't be here

File: plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java (hunk 2 above)
Comment (funkychicken493) [reply plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java]: Understood. I thought that deprecated would've been the right wording, will change.

File: plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java (hunk 3 above)
Comment (funkychicken493) [reply plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java]: When I made this change I was thinking that for most users, "file" and "disk" were synonymous, I will revert this.

File: plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java (hunk 4 above)
Comment (funkychicken493) [reply plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java]: I personally disagree, but I suppose I am following your formatting rules within this project, so I will revert the change.

File: plugin/src/main/java/com/denizenscript/denizen/utilities/command/NPCCommandHandler.java (hunk 1 above)
Comment (funkychicken493) [reply plugin/src/main/java/com/denizenscript/denizen/utilities/command/NPCCommandHandler.java]: I forgot about the possibility of color coding, so I will reset it after the name.

**Comment by funkychicken493** [comment]
requested changes have been made @mcmonkey4eva

File: plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java [hunk 5]
```diff
@@ -329,15 +329,15 @@ else if (args.getString(1).equalsIgnoreCase("scripts")) {
                 if (ScriptHelper.hadError()) {
                     Messaging.sendError(sender, "There was an error loading your scripts, check the console for details!");
                 }
-                Messaging.sendError(sender, "'/denizen reload scripts' is the old way of doing things ... use '/ex reload' instead!");
+                Messaging.sendError(sender, "'/denizen reload scripts' is functional but outdated... try using '/ex reload' instead!");
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java]: This should just be left the way it was

File: plugin/src/main/java/com/denizenscript/denizen/utilities/command/NPCCommandHandler.java [hunk 6]
```diff
@@ -243,7 +242,7 @@ else if (args.hasValueFlag("remove")) {
                 }
             }
             else {
-                Messaging.sendError(sender, "That NPC was already not assigned that script.");
+                Messaging.sendError(sender, npc.getName() + " was already not assigned to " + script + ".");
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/utilities/command/NPCCommandHandler.java]: needs color reset

**Pull request merged**